app.js

```diff
@@ -752,9 +762,9 @@ d.run(() => {
 			// Receives a 'cleanup' signal and cleans all modules
 			process.once('cleanup', error => {
 				if (error) {
-					scope.logger.fatal(error.toString());
+					appLogger.fatal(error.toString());
```

Comment:
I am not sure why we have specifics here, everywhere are logging `error` and only here I see converting to a string, not a huge concern, just matter of consistency.